src/core.js

```diff
@@ -820,6 +823,7 @@ goog.require('olcs.core.OlLayerPrimitive');
    */
   olcs.core.olPointGeometryToCesium = function(geometry, projection, style,
       billboards, opt_newBillboardCallback) {
+    var altitudeMode = geometry.get('altitudeMode');
```

Comment:
Since the code sprint at Schladming, Ol3 is able to parse the `altitudeMode` from KML geometries and set them as a geometry properties of Ol3 features.

In this PR I simply read this property and honour the requested mode.

I agree it would be great in the future to also handle `altitudeMode` at the layer level but it is not possible with the current design of the converters. I think we can add it later.